research/slim/BUILD

```diff
@@ -216,7 +216,7 @@ py_library(
     srcs = ["nets/alexnet.py"],
     srcs_version = "PY2AND3",
     deps = [
-        # "//tensorflow"
+        # "//tensorflow",
```

Comment:
Looks like you pulled in some commits from research/slim into this PR. Can you take those out?

Reply:
It turns out this stems from me making a commit on this branch before all of my git setup was complete. Should be resolved now.